pkg/doc/jwt/jwt.go

```diff
@@ -0,0 +1,448 @@
+/*
+ * Copyright SecureKey Technologies Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"reflect"
+	"strings"
+)
+
+const (
+	// HeaderB64 defines b64 header of boolean type
+	HeaderB64 = "b64" // bool
+
+	// HeaderType defines typ header of string type
+	HeaderType = "typ" // string
+
+	// HeaderAlgorithm defines alg header of string type
+	HeaderAlgorithm = "alg" // string
+
+	// HeaderKeyID defines kid header of string type
+	HeaderKeyID = "kid" // string
+
+	// TypeJWT defines JWT type
+	TypeJWT = "JWT"
+
+	// AlgorithmNone used to indicate unsecured JWT
+	AlgorithmNone = "none"
+
+	// SignatureEdDSA defines EdDSA alg
```

Comment:
Once both signer and verifier are externalized, should these constants be moved to the test files?

(Other than none, of course)

Reply:
They are used in `basicVerifier` implementation of `Verifier`. I think we should move these constants to the `verifier.go`. And probably make them internal and they won't be used externally.

Reply:
Changed.